### PR TITLE
Update Active_Directory_-_Get_User_Manager_Email_README.md

### PR DESCRIPTION
**Pull request by tschanfeld**
changed from ## to ### for items listed under dependencies

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Active_Directory_Query/Playbooks/Active_Directory_-_Get_User_Manager_Email_README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@ Returns the email address of the user's manager of an email address or a usernam
 ## Dependencies
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
-## Sub-playbooks
+#3# Sub-playbooks
```
Comment (barchen1) [inline Packs/Active_Directory_Query/Playbooks/Active_Directory_-_Get_User_Manager_Email_README.md:6]:
```suggestion
### Sub-playbooks
```

**Review by barchen1 — CHANGES_REQUESTED** [review]

**Review by barchen1 — APPROVED** [review]

**Review by tschanfeld — COMMENTED** [review]
oops, typo. thanks for catching that.

**Comment by tschanfeld** [comment]
@barchen1 looks like this got stuck, can you assist?

**Comment by barchen1** [comment]
Your changes is already in the file
Closing this PR.

**Pull request closed without merging**